dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java

```diff
@@ -65,7 +65,7 @@ public static String encrypt(String dataStr) {
       //todo:how to deal env
       if (labelsMap.containsKey("app") && labelsMap.get("app")
           .equals(KieConfig.INSTANCE.getAppName())
-          && labelsMap.containsKey("env") && labelsMap.get("env")
+          && labelsMap.containsKey("environment") && labelsMap.get("environment")
```

Comment:
Too many literals in a function, better to define constants for them.

Reply:
done